Crash when adding new folder

Application crashes on new folder creation.
Here is logcat:

```
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #2
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime: Process: com.nutomic.syncthingandroid.debug, PID: 4759
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime: java.lang.RuntimeException: An error occured while executing doInBackground()
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at android.os.AsyncTask$3.done(AsyncTask.java:300)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.FutureTask.run(FutureTask.java:242)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.lang.Thread.run(Thread.java:841)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:  Caused by: java.lang.ArrayIndexOutOfBoundsException: length=3; index=3
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at com.nutomic.syncthingandroid.syncthing.PostConfigTask.doInBackground(PostConfigTask.java:46)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at com.nutomic.syncthingandroid.syncthing.PostConfigTask.doInBackground(PostConfigTask.java:19)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at android.os.AsyncTask$2.call(AsyncTask.java:288)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.FutureTask.run(FutureTask.java:237)
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231) 
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
10-19 22:42:13.613 4759-7812/com.nutomic.syncthingandroid.debug E/AndroidRuntime:     at java.lang.Thread.run(Thread.java:841)
```

--- a/src/main/java/com/nutomic/syncthingandroid/syncthing/PostConfigTask.java
+++ b/src/main/java/com/nutomic/syncthingandroid/syncthing/PostConfigTask.java
@@ -43,8 +43,8 @@ public class PostConfigTask extends AsyncTask<String, Void, Boolean> {
         post.addHeader(new BasicHeader(RestApi.HEADER_API_KEY, params[1]));
 
         try {
-            post.setEntity(new StringEntity(params[3], HTTP.UTF_8));
-            Log.v(TAG, "API call parameters: " + params[3]);
+            post.setEntity(new StringEntity(params[2], HTTP.UTF_8));
+            Log.v(TAG, "API call parameters: " + params[2]);
             httpclient.execute(post);
         } catch (IOException|IllegalArgumentException e) {
             Log.w(TAG, "Failed to call Rest API at " + fullUri, e);

